explorer/client/src/utils/api/rpcSetting.ts

```diff
@@ -37,3 +37,19 @@ export const tryGetRpcSetting = (): string | null => {
     // query param takes precedence over local store
     return queryParam ? queryParam : localStore;
 };
+
+const ENDPOINTS = {
```

Comment:
this is mostly moving variables around from DefaultRpcClient, we might as well just put all this in the same file